src/inject.js

```diff
@@ -44,6 +45,24 @@ export default function injectDynamicImport(acorn) {
         return nextMethod.call(this, refDestructuringErrors);
       }
     ));
+
+    instance.extend('parseExpression', nextMethod => (
+      function parseExpression() {
+        const node = nextMethod.call(this);
+        if (node.type === 'CallExpression' && node.callee.type === 'Import') {
+          if (node.arguments.length !== 1) {
+            this.raise(node.start, 'import() requires exactly one argument');
+          }
+
+          const importArg = node.arguments[0];
+          if (importArg && importArg.type === 'SpreadElement') {
+            this.raise(importArg.start, '... is not allowed in import()');
+          }
+        }
+
+        return node;
+      }
+    ));
```

Comment:
Do you know where this will throw the token indicator?
I went a different (maybe more spec compliant route?) of not processing it as a `CallExpression` and treating it [as just an `AssignmentExpression`](https://tc39.github.io/proposal-dynamic-import/#prod-ImportCall). See [here](https://github.com/standard-things/esm/blob/19add243465e4131d4d89508a85f55fb18bcb769/src/acorn-extensions/dynamic-import.js#L43). That shakes out only a single argument and no spread support naturally (as is intended)